### ISSUE TITLE
Course facilitator role

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/london-money#25]
See [Description of course facilitator role](https://github.com/foundersandcoders/master-reference/issues/133#issuecomment-292799643). I would like us to consider whether we can afford to pay for this role.

**Comment by sofer** [comment]
I am strongly of the opinion that from an operational point of view, this does need to be a paid role and i am keen to proceed relatively swiftly. I am happy to make a case for it financially. What information would the finance committee need to allow us to decide whether this is a sensible decision?

**Comment by rebeccaradding** [comment]
@sofer are we also discussing how to make sure this role is paid in Nazareth next course?

**Comment by markwilliamfirth** [comment]
@sofer what's your view on the budget for this?

The NI primary threshold for 2017 is £8,164
The London Living Wage is £9.75. The annual equivalent for a part-time 20h/week role is £10,140
The 2017 personal allowance is £11,500

I'd suggest that one of these figures might be an appropriate place to start from and can be implemented quickly (implementing the NI primary threshold requires hardly any admin at all). The remuneration can then continue to be assessed.

**Comment by sofer** [comment]
@markwilliamfirth thank you so much for your response.

How would you feel about doing this on a self-employed basis for now? Something like £4,000 per cohort would be close to the numbers your outline . It's too low really, for the work involved, but it's a starting point. 

@rebeccaradding Nazareth is a separate discussion, but I agree that it needs to be a paid role as soon as funds will allow.

**Comment by markwilliamfirth** [comment]
From the role description, it seems to me that this role might have work in between cohorts as well and at first appears that it might be suitable for an all-year-round part-time employee of Founders & Coders, as opposed to a contractor. Is there good reason to make it a contractor role?

Agreed that we need to start somewhere. When the role is advertised (and I think it should be, as it should be open to anyone who may want to apply - perhaps a separate discussion to this) it should be noted that the remuneration will be reviewed after a certain time period.

**Comment by sofer** [comment]
@markwilliamfirth I can see you are as keen on putting this through PAYE as I am not. My reasons are primarily about tax efficiency and also about uncertainty about the role itself, but I am happy to be persuaded. At the level of income you are talking about, it makes no difference and once we are clearer about the role and can afford more, it may make sense to put somebody on the payroll anyway.

**Comment by markwilliamfirth** [comment]
I agree that it would be helpful to gain a clearer understanding of the role and thoroughly define the role requirements before moving forward discussing the financial plan for it. I think it's be wise to write and agree on a JD first.

**Comment by sofer** [comment]
@markwilliamfirth The [job description](https://github.com/foundersandcoders/master-reference/issues/133) is taking shape nicely. I feel like enough work is being done on it that we can continue to discuss the financial side in parallel.

Also,  two other arguments in favour of self-employment which I forgot in my earlier comment are:
1. the course-facilitator is likely to also be receiving income from client work through FAC--and trying to manager freelance payments and PAYE in parallel looks to me like a recipe for confusion;
2. it is not yet clear that the course facilitator role is something that one person should do for an extended period or whether it is something that should be done for a cohort or two before moving onto other work either within FAC or elsewhere.

I am in favour of offering £2,000 to facilitate the second half of FAC10 *now* and then reviewing all of this before the start of FAC11. If this is treated as a 50%-time role, then it is the equivalent of £24,000 p.a. pro rata, which feels to me like the minimum reasonable fee for this work.

We will be able to afford this level of expenditure based on our budget projections made at the beginning of the month, but I will be updating the numbers soon.

**Comment by jsms90** [comment]
@sofer Can we make this a proposal for our next meeting?

**Comment by jsms90** [comment]
In all honesty, I believe that our suggestion that this could be a part-time role is purely down to:
+ me only being able to dedicate 2-3 days per week while working for IP Cortex
+ @bradreeder intending to work full-time, whilst accepting part-time payment
+ our joint desperation for this role to be paid _at all_, so that FAC had someone dedicated to it
+ @des-des' desire to continue being a developer (very sorry if that's a misrepresentation Eoin!)

@bradreeder @des-des Please do weigh in if you believe any of the above is incorrect!

In all honesty, what I was capable of doing in 2-3 days per week was not enough. Way down the line, it may be entirely possible for a "course coordinator" / "course facilitaor" to focus on curriculum development. In which case, if that were the limit of the role, 20 hours per week _might_ be more reasonable.

Our current reality (and maybe the model we want for the future) is such that the course faciliator does much more than focus on the contents of the curriculum. We have heavy involvement in pastoral care, community outreach, graduate projects, and essentially provide interaction between all the moving parts in a campus.

As such, the role seems to be more of an overall campus representative.

In my opinion, this is not a part-time role.

**Comment by markwilliamfirth** [comment]
I agree with @jsms90 

Eventually each campus needs a full-time employee (that basically does a bit of everything in addition to leading the course), it's just a matter of when this should happen.

**Comment by jsms90** [comment]
@markwilliamfirth Cool :+1: 
Just wanted to clarify, since everything Bradley and I have said online makes it look like we consider part-time to be adequate, and this seemed to be the way we've left it with the Finance Committee
> If this is treated as a 50%-time role, then it is the equivalent of £24,000 p.a. pro rata, which feels to me like the minimum reasonable fee for this work.

For the sake of full transparency, yesterday, @iteles suggested that the current perception is that it be considered a full-time role, in the same way as it is in Gaza. But we understand that FAC does not yet have sufficient funds for comparable payment.

I believe the plan is for @bradreeder and I to supplement our income with the [Digital Co-ops Research Project](https://github.com/foundersandcoders/london-programme/issues/237) for the first 2 months.

So is the proposal actually to pay £24,000 p.a. pro rata for the 2nd half? i.e. £4000 over September-October?

**Comment by rebeccaradding** [comment]
@jsms90 the plan for July and August is to provide £2400 monthly compensation to London and Nazareth coordinators, equivalent to the Gaza coordinator. Given our current cashflow challenges, the London/Nazareth compensation will come jointly from FAC and from the Outlandish project. While we understand this is not ideal, it buys us time while we wait for the results of our Nazareth funding application and for London cashflow to increase (as it appears to be doing as a result of hiring fees).

The plan is to continue compensating course coordinators at this level, but we will revisit the question of _where_ that funding comes from in August once we have more clarity around funding in Nazareth and London.

**Comment by jsms90** [comment]
@rebeccaradding Gotcha. So are Bradley and I invoicing Outlandish directly for that half? Is Outlandish paying FAC, who is paying us? Does FAC sort this out with Outlandish or do we?

@rebeccaradding @iteles @sofer What are we discussing / proposing / voting on at the Finance Committee's Monday meeting?
Or is this just an "update from International" for everyone in the Finance Committee to be aware of?

**Comment by bradreeder** [comment]
@jsms90 I agree and that's a decent summary of my position. 

A large part of why I think it's difficult to scope this at the moment is that each campus has no paid staff, so all the things me and Jen indicated need doing gets lumped up into one role. Ideally I think we need to aim towards having a small team per campus, so that they're able to provide each other support and bounce ideas off each other.

tl;dr version is that this organisation has a lot of documentation and process holes and a lot of things that need to be worked out. It'd likely take a lot less time if all of these things were streamlined and solved, but until then it's basically the facilitator's role to work it out.

@rebeccaradding Thanks for the info.

**Comment by sofer** [comment]
Agreed to make £1,200/month available for London course facilitator for the next two months.

**Comment by markwilliamfirth** [comment]
@markwilliamfirth to send Dan dwyl freelancer contract with appropriate termination clause for F&C use
